src/main/java/org/octopusden/octopus/stringutils/StringUtils.java

```diff
@@ -11,9 +11,11 @@ public static String fillChar(char c, int count) {
   }
 
   /**
-   * Parse command line params ( <paramName>=<ParamValue>)
+   * Parse command line params ( &lt;paramName&gt;=&lt;ParamValue&gt;)
```

Comment:
Seems garbage. Please revert.

Reply:
Not garbage, unfortunately. It's not allowed to use xml tag symbols in javadoc comments, see https://stackoverflow.com/questions/2898897/how-can-i-use-and-in-javadoc-without-formatting

Reply:
Noted.